flink-container/docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Apache Flink cluster deployment on docker using docker-compose
+
+## Installation
+
+Install the most recent stable version of docker
+https://docs.docker.com/installation/
+
+## Build
+
+Images are based on the official Java Alpine (OpenJDK 8) image. If you want to
+build the flink image run:
+
+    sh build.sh --job-jar /path/to/job/jar/job.jar --image-name flink:job
```

Comment:
nit: The shebang in `build.sh` already tells the program loader to use `sh`.

Reply:
True. I will remove it.